charts/brokencrystals/values.yaml

```diff
@@ -1,7 +1,7 @@
 ingress:
-  url: brokencrystals.dev.vuln.nexploit.app
+  url: k3s.brokencrystals.nexploit.app
   cert: ""
   authlevel: "."
 images: 
-  main: latest
+  main: experimental
```

Comment:
I think it should be latest. Because "experimental" version is for "without a proxy" version of BC

Reply:
Yes, indeed.